### PR TITLE
MSSB-41: Main menu tabbing

### PR DESCRIPTION
**Pull request by CocoaBean5**
## JIRA Ticket(s)

- See ticket https://palantir.atlassian.net/browse/MSSB-41

## Description

- Adjusted JS and CSS to include main nav and children items to be accessible via keyboard tabbing.

BEFORE:
Screen Shot 2020-01-20 at 9 19 52 AM

AFTER:
Screen Shot 2020-01-20 at 3 27 33 PM

## Testing instructions

- `cd web/themes/custom/blrfurther_d8`
- `gulp sass`
- Tab through a page and make sure the main nav items are accessible.

### CODE REVIEW

File: web/themes/custom/blrfurther_d8/assets/css/partials/_nav.scss [hunk 1]
```diff
@@ -84,98 +95,103 @@
...
-			&__item{
-				@include position_relative();
-				display: block;
-				float: none;
-				text-align: left;
+
+			.menu {
+				display: table;
+				height: auto;
+				//padding-top: 20px;
```
Comment (russom-woldezghi) [inline web/themes/custom/blrfurther_d8/assets/css/partials/_nav.scss]: Remove commented out `padding-top`.

File: web/themes/custom/blrfurther_d8/assets/js/menu.js [hunk 2]
```diff
@@ -12,35 +12,40 @@ jQuery(document).ready(function($){
...
+	 */
+	if( $('.layout__sidebar.-first .navigation.-primary.-two').length ){
+		// do nothing
 	}
 	else{
-		$('.layout__sidebar.-first').addClass('-active');
+		// remove the left nav and content class
+		//$('.layout__sidebar.-first').detach();
+		//$('.layout__content').removeClass('-sidebar_first');
 	}
```
Comment (russom-woldezghi) [inline web/themes/custom/blrfurther_d8/assets/js/menu.js]: This can probably be removed.

**Comment by CocoaBean5** [comment]
The comments in code are from the previous vendor. @russom-woldezghi Do you think I should remove them? Or leave them for possible context later?

File: web/themes/custom/blrfurther_d8/assets/js/menu.js (hunk 2 above)
Comment (russom-woldezghi) [reply web/themes/custom/blrfurther_d8/assets/js/menu.js]: @CocoaBean5 The above if/else should be removed. We can leave the reset for now.

**Comment by megansabina** [comment]
@CocoaBean5  - I think it's fine to leave the comments, but you can remove the empty if / else

**Comment by CocoaBean5** [comment]
@megansabina The code has been removed.
